plotting/hovmoller.py

```diff
@@ -113,39 +85,20 @@ def find_depth(depth, clip_length, dataset):
         # Load data sent from Right Map (if in compare mode)
         if self.compare:
             compare_config = DatasetConfig(self.compare['dataset'])
-            with open_dataset(compare_config, variable=self.compare['variables']) as dataset:
+            with open_dataset(compare_config, timestamp=self.compare['starttime'], endtime=self.compare['endtime'], variable=self.compare['variables']) as dataset:
```

Comment:
This will resolve a crash when trying to open the compared dataset. No other testing done beyond this since it's outside the scope of this PR.